ui/app/adapters/pki/tidy.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import { assert } from '@ember/debug';
+import { encodePath } from 'vault/utils/path-encoding-helpers';
+import ApplicationAdapter from '../application';
+
+export default class PkiTidyAdapter extends ApplicationAdapter {
+  namespace = 'v1';
+
+  urlForCreateRecord(snapshot) {
+    const { type, backend } = snapshot.record;
```

Comment:
I made some comments below, but I think this might make more sense to pass as `adapterOptions` instead of a param on the model 🤔